centreon/www/include/configuration/configObject/contact/listContact.php

```diff
@@ -353,11 +365,13 @@
         "RowMenu_badge" => $contact["contact_activate"] ? "service_ok" : "service_critical",
         "RowMenu_refreshLdap" => $isLinkedToLdap ? $refreshLdapBadge[$isLinkedToLdap] : "",
         "RowMenu_refreshLdapHelp" => $isLinkedToLdap ? $refreshLdapHelp[$isLinkedToLdap] : "",
-        "RowMenu_options" => $moptions
+        "RowMenu_options" => $moptions,
+        "RowMenu_unblock" => $contact["blocking_time"] ? $blocked_user_icon : "-"
```

Comment:
```suggestion
        "RowMenu_unblock" => $contact["blocking_time"] !== null ? $blocked_user_icon : "-"
```
to avoid comparison with 0